tests/PoExtractorTest.php

```diff
@@ -47,8 +47,8 @@ public function testAutomaticHeaders()
         //Extract po with no language/domain headers
         $translations = Gettext\Extractors\Po::fromFile(__DIR__.'/files/plurals.po');
 
-        $this->assertNull($translations->getLanguage(), 'Something erroneously set for language');
-        $this->assertNull($translations->getDomain(), 'Something erroneously set for domain');
+        $this->assertEmpty($translations->getLanguage(), 'Something erroneously set for language');
```

Comment:
I think they should return null if the header is not defined and empty string if its defined but emptied